src/app/space-settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -62,4 +70,12 @@ export class CreateAreaDialogComponent implements OnInit, OnDestroy {
   cancel() {
     this.host.close();
   }
+
+  handleError(error: any) {
+    if (error.errors[0].detail.indexOf('unique') > -1) {
```

Comment:
Do you want me to just update the code so we show a generic error until the API is updated to have a more specific error?